Dockerfile

```
@@ -34,14 +34,14 @@ USER $NB_UID
 
 # install rOMERO
 ENV _JAVA_OPTIONS="-Xss2560k -Xmx2g"
```

Needs an additional environment variable:
```suggestion
ENV OMERO_LIBS_DOWNLOAD=TRUE
```
This will tell the `romero.gateway` package to go ahead and download the Java libs without asking the user. Since they are not included any longer, the user has to confirm the download when the package is loaded for the first time.